### PR TITLE
Updated to reCAPTCHA 2, added a new style, and fixed a JS bug.

### PR DESCRIPTION
**Pull request by Circlepuller**
- recaptchalib.php was outdated and ineffective.
- sharp.css is a nice, fairly clean theme.
- jQuery needed to be loaded before inline-expanding.js which is enabled in default installations.

Also, it's been a hell of a long time since I've made changes to anything TinyBoard, haha.

### CODE REVIEW

**Comment by Circlepuller** [comment]
Okay, there was also an edit I had to make to allow checkSpam to work correctly in config.php
https://github.com/Circlepuller/Tinyboard/commit/a31a3a281fde71b3ca50f495dc5f98ee537a8d80

File: post.php [hunk 1]
```diff
@@ -383,14 +383,15 @@
...
 				error($config['error']['bot']);	
 				// Check what reCAPTCHA has to say...
-				$resp = recaptcha_check_answer($config['recaptcha_private'],
-				$_SERVER['REMOTE_ADDR'],
-				$_POST['recaptcha_challenge_field'],
-				$_POST['recaptcha_response_field']);
-			if (!$resp->is_valid) {
+			$resp = json_decode(file_get_contents(sprintf('https://www.google.com/recaptcha/api/siteverify?secret=%s&response=%s&remoteip=%s',
+				$config['recaptcha_private'],
+				$_POST['g-recaptcha-response'],
```
Comment (czaks) [inline post.php]: this line is exploitable

Comment (Circlepuller) [inline post.php]: Do you mean the request, or `$resp->is_valid`? In the case of the latter, I just changed it to `$resp['success']` which is version 2's success variable.

Comment (Circlepuller) [inline post.php]: I also used urlencode on the user input for safety, if that helps.
https://github.com/Circlepuller/Tinyboard/commit/18aed77ecc7995a08e73507d5d6733bbe2f36dcb

**Pull request merged**